.gitignore

```diff
@@ -0,0 +1,4 @@
+*.py[co]
+.eggs/
+.tox/
+*.egg-info/
```

Comment:
This looks fine...maybe just put it in a separate PR, and we can just merge it immediately?